backend/person/models.py

```diff
@@ -57,10 +57,12 @@ class PersonReference(Field, models.Model):
     person = models.ForeignKey(
         to=HistoricalPerson,
         on_delete=models.CASCADE,
+        related_name="person_references",
```

Comment:
For the HistoricalPerson model, `description_references` may be more appropriate?

Reply:
Agreed!